pkg/activator/handler/handler.go

```diff
@@ -173,30 +113,26 @@ func (a *activationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	tryStart := time.Now()
-	err = a.throttler.Try(tryContext, revID, func() {
+	err := a.throttler.Try(tryContext, revID, func(dest string) error {
 		trySpan.End()
 		a.logger.Debugf("Waiting for throttler took %v time", time.Since(tryStart))
 
-		probeCtx, probeSpan := trace.StartSpan(r.Context(), "probe")
-		success, attempts := a.probeEndpoint(logger, r.WithContext(probeCtx), target)
-		probeSpan.End()
-
 		var httpStatus int
-		if success {
-			// Once we see a successful probe, send traffic.
-			attempts++
-			proxyCtx, proxySpan := trace.StartSpan(r.Context(), "proxy")
-			httpStatus = a.proxyRequest(w, r.WithContext(proxyCtx), target)
-			proxySpan.End()
-		} else {
-			httpStatus = http.StatusInternalServerError
-			w.WriteHeader(httpStatus)
+		target := url.URL{
+			Scheme: "http",
+			Host:   dest,
 		}
-
-		configurationName := revision.Labels[serving.ConfigurationLabelKey]
-		serviceName := revision.Labels[serving.ServiceLabelKey]
-		a.reporter.ReportRequestCount(namespace, serviceName, configurationName, name, httpStatus, attempts, 1.0)
+		// Once we see a successful probe, send traffic.
```

Comment:
Why is `probeEndpoint(...)` removed? Polling is done in revision backend manager's `run`, which calls [`checkDests()`](https://github.com/knative/serving/blob/63fc7bbe26fad6b5a328aaa7bcb1f48cb89800aa/pkg/activator/net/revision_backends.go#L209) to probe cluster IP and pod IPs, but polling right before sending traffic is still necessary, right?